partcad/tests/unit/test_genai.py

```
@@ -0,0 +1,68 @@
+
+
+def check_python_syntax(python_code):
+    try:
+        compile(python_code, "<string>", "exec")
+        return True
+    except SyntaxError as e:
+        print(f"Syntax error detected:\n{e}")
+        return False
+
+
+def test_scad_ai_script_header():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_scad_ai_script_header`

Add return type annotation: `None`

(ANN201)